fedimint-server-core/src/bitcoin_rpc.rs

```diff
@@ -27,9 +29,16 @@ impl ServerBitcoinRpcMonitor {
         let (status_sender, status_receiver) = watch::channel(None);
 
         let rpc_clone = rpc.clone();
+        debug!(
+            target: LOG_SERVER,
+            interval_ms  = %update_interval.as_millis(),
+            "Starting bitcoin rpc monitor"
+        );
 
         task_group.spawn_cancellable("bitcoin-status-update", async move {
+            let mut interval = tokio::time::interval(update_interval);
             loop {
+                interval.tick().await;
```

Comment:
Minimal semantic change, but I like it
![Screenshot 2025-05-09 at 12-03-34 interval in tokio time - Rust](https://github.com/user-attachments/assets/cca0f0bf-ba85-4cfa-9d8b-4640596354e3)

Reply:
Yeah, interval is generally always better for periodic things.